apps/heft/src/plugins/TypeScriptPlugin/TypeScriptBuilder.ts

```diff
@@ -147,7 +147,9 @@ export class TypeScriptBuilder extends SubprocessRunnerBase<ITypeScriptBuilderCo
         .replace(/\+/g, '-')
         .replace(/\//g, '_');
 
-      this.__tsCacheFilePath = `${this._configuration.buildMetadataFolder}/ts_${serializedConfigHash}.json`;
+      // Paranoia. If backslashes sneak it in breaks incremental compilation
```

Comment:
I think you meant "in it" not "it in".
Also, we can probably remove the "Paranoia" part. 😁

Reply:
Fixed.